cloud_governance/main/environment_variables.py

```diff
@@ -43,7 +43,7 @@ def __init__(self):
                                                                         'nat_gateway_unused',
                                                                         'zombie_snapshots', 'skipped_resources',
                                                                         'monthly_report']
-        self._environment_variables_dict['aws_cost_policies'] = ['cost_explorer', 'cost_over_usage']
+        self._environment_variables_dict['aws_cost_policies'] = ['cost_explorer', 'cost_over_usage', 'cost_explorer_reports']
```

Comment:
cost_explorer_reports, is it related to forecast ?
If yes, you should choose name like cost_explorer_forecast_reports.

Reply:
cost_explorer_reports is combination of Budget, Forecast, acutaul
So I named as cost_explorer reports